.github/workflows/module.build.yml

```diff
@@ -195,6 +195,70 @@ jobs:
             labs: false
             skip: false
 
+          # Test: Bazel 4
+          - label: Bazel 4
+            target: sample
+            action: build
+            directory: "./example/integration_tests/bazel4"
+            labs: false
+            skip: ${{ contains(inputs.runner, 'windows') }}
+
+          # Test: Bazel 5
+          - label: Bazel 5
+            target: sample
+            action: build
+            directory: "./example/integration_tests/bazel5"
+            labs: false
+            skip: ${{ contains(inputs.runner, 'windows') }}
+
+          # Test: Bazel 6
+          - label: Bazel 6
+            target: sample
+            action: build
+            directory: "./example/integration_tests/bazel6"
+            labs: false
+            skip: false
+
+          # Test: Inert
+          - label: Inert
+            target: sample
+            action: build
+            directory: "./example/integration_tests/inert"
+            labs: false
+            skip: false
+
+          # Test: Inert Workspace
+          - label: Inert Workspace
+            target: sample
+            action: build
+            directory: "./example/integration_tests/inert-workspace"
+            labs: false
+            skip: false
```

Comment:
new "inert" workspace and bzlmod tests show that GraalVM is _not_ downloaded _unless_ there is a `native_image` target _or_ a toolchain is registered.